hrm-domain/hrm-core/case/caseService.ts

```diff
@@ -459,3 +460,29 @@ export const getCasesByProfileId = async (
     });
   }
 };
+
+export const searchCasesByIds = generalizedSearchCases(searchByCaseIds);
+
+export const searchCasesByIdCtx = async (
```

Comment:
Are both of this used in outside modules? If not, let's only `export` what's strictly necessary.